src/cli/index.ts

```diff
@@ -16,27 +19,40 @@ if (command === 'dev') {
       logTypeScriptMessage('Compiling TypeScript sources...')
     },
     onFileChanged() {
-      logLineWithIcon('⟳ ', 'File change detected. Starting incremental compilation...')
+      logLineWithIcon(
+        '⟳ ',
+        'File change detected. Starting incremental compilation...'
+      )
     },
     onCompilationFailed(message) {
       killDevChildProcess()
       logTypeScriptMessage(message)
     },
     onCompilationDone() {
       killDevChildProcess()
-      devChildProcess = spawn('node', ['./node_modules/@asyncapi/glee/dist/cli/start.js'], {
-        stdio: 'inherit',
-        env: {
-          ...process.env,
-          NODE_ENV: 'development',
-        },
-      })
-    }
+      devChildProcess = spawn(
+        'node',
+        ['./node_modules/@asyncapi/glee/dist/cli/start.js'],
+        {
+          stdio: 'inherit',
+          env: {
+            ...process.env,
+            NODE_ENV: 'development',
+          },
+        }
+      )
+    },
   })
 } else if (command === 'start') {
   import('./start.js')
+} else if (command === 'docs') {
+  (async () => {
+    const config = await initializeConfigs()
+    const parsedAsyncAPI = await getParsedAsyncAPI()
+    generateDocs(parsedAsyncAPI, config, null)
```

Comment:
It would be awesome if we could just have this:

```suggestion
    generateDocs()
```

Probably another function in another file called `src/docs.ts`. This way we hide the complexity from this file.